### PR TITLE
Add logging in commands

### PR DESCRIPTION
**Pull request by Ma-Xizi**

### CODE REVIEW

File: src/main/java/tutorease/address/logic/commands/DeleteContactCommand.java [hunk 1]
```diff
@@ -46,16 +51,19 @@ public DeleteContactCommand(Index targetIndex) {
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
+        logger.log(Level.INFO, "Executing Delete Contact Command");
         requireNonNull(model);
         List<Person> lastShownList = model.getFilteredPersonList();
 
         if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            logger.log(Level.WARNING, "Invalid target index: {0}", targetIndex.getOneBased());
```
Comment (Ma-Xizi) [inline src/main/java/tutorease/address/logic/commands/DeleteContactCommand.java:59]: In looginf, should we use oneBased index or zero based?

Comment (NorbertLoh) [reply src/main/java/tutorease/address/logic/commands/DeleteContactCommand.java:59]: i think maybe zero based as the check if for zero based

Comment (Ma-Xizi) [reply src/main/java/tutorease/address/logic/commands/DeleteContactCommand.java]: okay sure

File: src/main/java/tutorease/address/logic/commands/ListContactCommand.java [hunk 2]
```diff
@@ -13,14 +17,19 @@ public class ListContactCommand extends ContactCommand {
...
 
     @Override
     public CommandResult execute(Model model) {
+        logger.log(Level.INFO, "Executing List Contact Command");
         requireNonNull(model);
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
-        if (model.getFilteredPersonList().isEmpty()) {
+        assert model.getFilteredPersonList() != null;
+        if (model.getFilteredPersonListSize() == 0) {
+            logger.log(Level.INFO, "No contacts exist in TutorEase.");
```
Comment (Ma-Xizi) [inline src/main/java/tutorease/address/logic/commands/ListContactCommand.java:29]: I changed it slightly to align with Law of Dementer. Is this allowed?

**Pull request merged**